In OpenBSD, we do not compile programs statically when it isn't necessary.
Drop -Werror

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -1,7 +1,6 @@
 VERSION=1.11.2
 CFLAGS?=-g
-MAINFLAGS:=-DVERSION='"$(VERSION)"' -Wall -Wextra -Werror -Wno-unused-parameter
-LDFLAGS+=-static
+MAINFLAGS:=-DVERSION='"$(VERSION)"' -Wall -Wextra -Wno-unused-parameter
 INCLUDE+=-Iinclude
 PREFIX?=/usr/local
 BINDIR?=$(PREFIX)/bin
